src/test/kotlin/com/buildit/bookit/v1/bookable/BookableControllerTests.kt

```diff
@@ -12,26 +12,19 @@ import org.jetbrains.spek.api.dsl.on
 /**
  * Bookable controller unit tests
  */
-object BookableControllerTests : Spek(
-    {
-        describe("get known location")
-        {
-            on("GET")
-            {
-                it("should return UP")
-                {
+object BookableControllerTests : Spek({
+        describe("get known location") {
```

Comment:
minor, but these are bookables not locations

Reply:
done